### DISCUSSION TITLE
can I ignore these warnings and proceed with the next step of analysis using the assembly results?

### DISCUSSION THREAD

**Discussion by 799lucky** [ablab/spades#1412] [Q&A]
So, it's not a failure. These are warnings indicating that the assembly results might be suboptimal

_Originally posted by @asl in https://github.com/ablab/spades/issues/460#issuecomment-593276520_
            
So, when I encounter the following warnings, can I ignore these warnings and proceed with the next step of analysis using the assembly results? Thanks for your help.

=== Error correction and assembling warnings:

1:47:47.483 82G / 113G WARN General (kmer_coverage_model.cpp : 327) Valley value was estimated improperly, reset to 57
1:47:47.494 82G / 113G WARN General (kmer_coverage_model.cpp : 366) Failed to determine erroneous kmer threshold. Threshold set to: 57
2:34:50.652 81G / 112G WARN General (kmer_coverage_model.cpp : 327) Valley value was estimated improperly, reset to 16
2:34:50.659 81G / 112G WARN General (kmer_coverage_model.cpp : 366) Failed to determine erroneous kmer threshold. Threshold set to: 16
3:43:21.155 68G / 97G WARN General (kmer_coverage_model.cpp : 218) Too many erroneous kmers, the estimates might be unreliable
3:43:39.815 68G / 97G WARN General (kmer_coverage_model.cpp : 327) Valley value was estimated improperly, reset to 8
======= Warnings saved to /home/pmslab/Desktop/Raman/Convallaria_mt/keiskei_hybrid_assembly/warnings.log

**Comment by asl** [comment]
> So, when I encounter the following warnings, can I ignore these warnings and proceed with the next step of analysis using the assembly results? Thanks for your help.

It is only you who can decide if the assembly results are usable for the next step of analysis or not.

**Reply by 799lucky** [reply]
Thanks for your help
